cockatrice/src/SpoilerBackgroundUpdaterThread.cpp

```diff
@@ -0,0 +1,249 @@
+#include <QDateTime>
```

Comment:
This file needs to be renamed in order to be all lowercase, or compilation will fail on case sensitive filesystems (eg. Linux)